release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -19,7 +19,7 @@ Built on {op-system-base-full} and Kubernetes, {product-title} provides a more s
 {product-title} {product-version} clusters are available at https://console.redhat.com/openshift. With the {cluster-manager-first} application for {product-title}, you can deploy {product-title} clusters to either on-premises or cloud environments.
 
 // Double check OP system versions
-{product-title} {product-version} is supported on {op-system-base-full} 8.8-8.10, and on {op-system-first} 9.4.
+{product-title} {product-version} is supported on {op-system-base-full} 8.8 and a later version of {op-system-base} 8 that is released before End of Life (EoL) of {product-title} {product-version} and on {op-system-first} 4.17.
```

Comment:
Please follow the two-sentence note done for the 4.18 update: https://github.com/openshift/openshift-docs/pull/83914/files

Best to keep things consistent.